src/raftstore/store/worker/apply.rs

```diff
@@ -55,6 +55,7 @@ use super::metrics::*;
 
 const WRITE_BATCH_MAX_KEYS: usize = 128;
 const DEFAULT_APPLY_WB_SIZE: usize = 4 * 1024;
+const SHRINK_PENDING_CMD_QUEUE_CAP: usize = 64;
```

Comment:
Is the value suitable?

Reply:
This value is adjusted with SHRINK_CACHE_CAPACITY which is 64, i think there is at most one pending command relate to one entry?